Added documentation for PR #9026 (#118)
mautic/mautic-documentation: pages/09.contacts/01.manage-contacts/01.contact-overview/docs.en.md

The main tab displays the contact's action history sorted from the latest to the oldest. Each action will display as many details as there is about it. For example a form submission will display what values the contact submitted, the email send action will inform you whether the email was opened and when and so on. You can filter what actions you want to include or exclude from the timeline if you are searching for a specific one.

> A "Page hit" event type will also appear in the main history tab. If the page's title is written in non-Latin characters (e.g., Hebrew, Chinese, Arabic), the page title will appear transliterated in the history tab.

Reviewer: ```suggestion
A "Page hit" event type will also appear in the main history tab. If the page title is written in non-Latin characters (e.g., Hebrew, Chinese, Arabic), it will appear transliterated in the history tab.
```

> You can turn this off from Configuration > System Settings > Transliterate page titles.

Reviewer: ```suggestion
You will find the configuration to turn the transliteration On/Off under: Configuration > System Settings > Transliterate page titles.
```

> This setting will then show the original page title in the history tab.

Reviewer: ```suggestion
If transliteration is disabled, the original page titles will be shown in the history tab instead.
```
